resources/lib/monitor.py

```diff
@@ -69,7 +69,7 @@ def onNotification(self, sender, method, data):
         elif sender.startswith('upnextprovider'):
             method = method.split('.')[1]
 
-            if method not in ('plugin.video.emby_play_action'):
+            if method not in ('plugin.video.jellyfin_play_action'):
```

Comment:
Looks like they don't understand Python tuples:
```suggestion
            if method not in ('plugin.video.jellyfin_play_action',):
```